sympy/physics/mechanics/tests/test_lagrange.py

```diff
@@ -138,7 +138,7 @@ def test_dub_pen():
     q1dd, q2dd = dynamicsymbols('q1 q2', 2)
     u1, u2 = dynamicsymbols('u1 u2')
     u1d, u2d = dynamicsymbols('u1 u2', 1)
-    l, m, g = symbols('l m g')
+    l, m, g = symbols('l m g', real=True)
```

Comment:
@isuruf Though I'm not sure why this happens, but removing real=True argument fixes the failure for test_lagrange.py. This is probably because at the time of using sympify over these objects, the resulting test expression isn't simplified. As far as I know, the Symbol class in SymEngine.py does take extra arguments but nothing is done of them.